### PR TITLE
feat: Implement the prototype

### PR DESCRIPTION
**Pull request by gaocegege**
**What this PR does / why we need it**:

This PR contains:

- Prototype
- OWNERs change
- vendor change

The document will be in anther PR. Please review this commit.

https://github.com/caicloud/ormb/pull/1/commits/9a5f00a77a14d3d1977c7d6f3e7392d14da101a2

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @ddysher @codeflitting 

/cc @huangyingwen @zhujian7 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by ddysher** [comment]
@huangyingwen  is our frontend developer, i guess you want to @hyy0322 

/cc @supereagle

**Comment by gaocegege** [comment]
Yeah. Sorry for disturbing you.

**Comment by ddysher** [comment]
I've briefly scanned the source, the prototype LGTM. :zap: 

regarding docs, we might need design doc (with roadmap, integration plan with harbor, etc), as well as a quick-start doc.

**Comment by gaocegege** [comment]
Yeah I think so. 

This prototype is just used to illustrate the features. After the meeting with VMware I will

- Add quick start, design proposal, model config specification
- Add roadmap

**Comment by ddysher** [comment]
LGTM

File: pkg/consts/consts.go [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package consts
+
+const (
+	ModelContentLayerMediaType = "application/tar+gzip"
+	ModelConfigMediaType       = "application/vnd.caicloud.model.config.v1+json"
```
Comment (supereagle) [inline pkg/consts/consts.go:5]:
```suggestion
	MediaTypeModelContentLayer = "application/tar+gzip"
	MediaTypeModelConfig       = "application/vnd.caicloud.model.config.v1+json"
```

Comment (gaocegege) [reply pkg/consts/consts.go]: Addressed

**Comment by gaocegege** [comment]
Please take a look, thanks for your time :smile:

**Comment by supereagle** [comment]
LGTM

/cc @codeflitting  @huangyingwen @zhujian7

**Comment by codeflitting** [comment]
we should add gendoc to cmd for auto-generated documentation

**Comment by gaocegege** [comment]
Can we merge it now? It will be used in the meeting tomorrow.

**Comment by ddysher** [comment]
LGTM

/lgtm

**Pull request merged**